lhotse/cut.py

```diff
@@ -1976,6 +2015,7 @@ def reverb_rir(
         normalize_output: bool = True,
         early_only: bool = False,
         affix_id: bool = True,
+        keep_mono: bool = False,
```

Comment:
Can we make it `True` by default? For simple augmentation in scenarios like ASR / pre-training, it seems like the right thing to do.